[logs] document beforeSend API (#664)
DataDog/browser-sdk: packages/logs/README.md

## Advanced usage

### Scrub sensitive data from your Browser logs

> If your Browser logs data contains sensitive information that need redacting, configure the Browser SDK to scrub sensitive sequences by using the `beforeSend` callback when you initialize the Browser Log Collector.

Reviewer: ```suggestion
If your Browser logs contain sensitive information that needs redacting, configure the Browser SDK to scrub sensitive sequences by using the `beforeSend` callback when you initialize the Browser Log Collector.
```

> This callback function gives you access to every event collected by the Browser SDK before they get sent to Datadog.

Reviewer: ```suggestion
The `beforeSend` callback function gives you access to each event collected by the Browser SDK before it is sent to Datadog, and lets you update commonly redacted properties.
```

> For example, redact email addresses from your web application URLs:

Reviewer: ```suggestion
For example, to redact email addresses from your web application URLs:
```

window.DD_LOGS &&
    window.DD_LOGS.init({
        ...,
        beforeSend: (event) => {
            // remove email from view url
            event.view.url = event.view.url.replace(/email=[^&]*/, "email=REDACTED")
        },
        ...
    });
```

You can update the following event properties:

| Attribute       | Type   | Description                                                                                      |
| --------------- | ------ | ------------------------------------------------------------------------------------------------ |
| `view.url`      | String | The URL of the active web page.                                                                  |
| `view.referrer` | String | The URL of the previous web page from which a link to the currently requested page was followed. |
| `message`       | String | The content of the log.                                                                          |
| `error.stack`   | String | The stack trace or complementary information about the error.                                    |
| `http.url`      | String | The HTTP URL.                                                                                    |

> **Note**: The Browser SDK will ignore modifications made to event properties not listed above. Find out about all event properties on the [Browser SDK repository][5].

Reviewer: ```suggestion
**Note**: The Browser SDK will ignore modifications made to event properties not listed above. For more information about event properties, see the [Browser SDK repository][5].
```
